sdk/core/abort-controller/src/AbortController.ts

```diff
@@ -58,8 +58,24 @@ export class AbortError extends Error {
 export class AbortController {
   private _signal: AbortSignal;
 
-  constructor() {
+  /**
+   * @param {AbortSignal} [parentSignal] The AbortSignal that will signal aborted on the AbortSignal associated with this controller.
+   * @constructor
+   */
+  constructor(parentSignal?: AbortSignal) {
```

Comment:
This should have two overloads:

1. AbortSignal[]
2. ... args: AbortSignal[]

IOW, should be able to pass any number of parent signals or even an array of signals.

Reply:
Ah, I hadn’t considered supporting multiple parent signals, will change!

Reply:
Updated!